contrib/libpoco/Foundation/CMakeLists.txt

```diff
@@ -73,27 +69,18 @@ else()
         src/pcre_valid_utf8.c
         src/pcre_xclass.c
     )
-
-    # zlib
-    POCO_HEADERS( SRCS zlib
-        include/Poco/zconf.h
-        include/Poco/zlib.h
-        )
-
-    POCO_SOURCES( SRCS zlib
```

Comment:
I dropped zlib that was bundled with Poco from build (or else zlib symbols would be duplicated everywhere, leading to one of the two sets of symbols selected based on link order).